src/ShearStressTransportEquationSystem.C

```diff
@@ -411,10 +434,33 @@ ShearStressTransportEquationSystem::update_and_clip()
 void
 ShearStressTransportEquationSystem::clip_min_distance_to_wall()
 {
-  // if this is a restart, then min distance has already been clipped
-  if (realm_.restarted_simulation())
+  if (realm_.restarted_simulation() && !realm_.has_mesh_motion())
     return;
 
+#if 1
```

Comment:
Did you mean to leave `#if 1` in here?

Reply:
@alanw0 Seems like github showed you an older version of the commits? 
0e62f58 should have removed those `#if 1` statements.

Reply:
Hmm. That's weird. In any case, it's fine then. :-)